### PR TITLE
skip tests if MGR_CMD_DELETE or MGR_CMD_CREATE node is used on cpuset mom

### PR DESCRIPTION
**Pull request by visheshh**
skip tests if MGR_CMD_DELETE or MGR_CMD_CREATE node is used on cpuset mom

#### Describe Bug or Feature
Some tests directly call delete node or create node instead of create_vnodes. In that case tests are not skipped on cpuset mom

#### Describe Your Change
skip tests if MGR_CMD_DELETE or MGR_CMD_CREATE node is used on cpuset mom

#### Attach Test and Valgrind Logs/Output

[smokeprov.txt](https://github.com/openpbs/openpbs/files/5908870/smokeprov.txt)
[provenh.txt](https://github.com/openpbs/openpbs/files/5908871/provenh.txt)

### CODE REVIEW

File: test/fw/ptl/lib/ptl_server.py [hunk 1]
```diff
@@ -2251,7 +2269,13 @@ def manager(self, cmd, obj_type, attrib=None, id=None, extend=None,
         :raises: PbsManagerError
         """
 
-        if cmd == MGR_CMD_SET and id is not None and obj_type == NODE:
+        if cmd == MGR_CMD_DELETE and obj_type == NODE:
+            for cmom, momobj in self.moms.items():
+                if momobj.is_cpuset_mom():
+                    self.skipTest("Do not delete nodes on cpuset moms")
+
+        if (cmd == MGR_CMD_SET or cmd == MGR_CMD_CREATE and
```
Comment (birbalsain) [inline test/fw/ptl/lib/ptl_server.py:2277]: But you are not skipping for cmd==MGR_CMD_CREATE and obj_type == NODE

**Review by bayucan — APPROVED** [review]
Looks good.

**Review by birbalsain — APPROVED** [review]
Changes LGTM

**Comment by visheshh** [comment]
@bayucan Please take this PR forward.

**Comment by hirenvadalia** [comment]
@visheshh please resolve conflicts

**Comment by visheshh** [comment]
@hirenvadalia as function had been moved to ptl_wrapper.py. I have redone the changes and force pushed.

**Review by bayucan — APPROVED** [review]
Still looks good.

**Comment by visheshh** [comment]
@hirenvadalia resolved conflicts. By reverting change to ptl_server.py as the original change for ptl_wrappers.py was reverted.

**Pull request merged**